test/DynamoCoreTests/Models/DynamoModelEventsTest.cs

```diff
@@ -365,14 +366,20 @@ public void TestOnRequestsCrashPrompt()
             CurrentDynamoModel.RequestsCrashPrompt += CurrentDynamoModel_RequestsCrashPrompt;
             var crashArgs = new Dynamo.Core.CrashPromptArgs("Crash Event", "Test Message");
 
+            var e = new Exception("Test");
+            var cerArgs = new CrashErrorReportArgs(null, e);
+            var cArgs = new Dynamo.Core.CrashPromptArgs(e);
+
             //Act
+            CurrentDynamoModel.OnRequestsCrashPrompt(null, cerArgs);
+            CurrentDynamoModel.OnRequestsCrashPrompt(null, cArgs);
             CurrentDynamoModel.OnRequestsCrashPrompt(this, crashArgs);
 
             //Assert
             //Unsubcribe from event
             CurrentDynamoModel.RequestsCrashPrompt -= CurrentDynamoModel_RequestsCrashPrompt;
             //This will validate that the local handler was executed and set the flag in true
-            Assert.IsTrue(requestsCrashPrompt);
+            Assert.AreEqual(3, requestsCrashPrompt);
```

Comment:
should the test reset `requestsCrashPrompt`?

Reply:
teardown / setup should already be doing cleanup